src/templates/en/2019/base_chapter.html

```diff
@@ -18,7 +18,7 @@
   indexBox.classList.toggle('show');
 });
 
-fetch(`https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json`)
+fetch("https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json")
```

Comment:
The string template `{{ ... }}` only works with backticks. Either revert this or use string concatenation. 
```suggestion
fetch(`https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json`)
```

Reply:
Are you sure? Just tested again and definitely seems to work...

Reply:
Oh no you're right. I was confusing this with the `${var}` syntax in JS. The `{{ ... }}` are replaced by the Jinja renderer in Python.

Reply:
Phewww! Thought you'd caught be out pushing untested code again - but not this time!! ☺️